{{cookiecutter.repo_name}}/setup.py

```diff
@@ -29,7 +29,7 @@
     long_description_content_type="text/markdown",
     version=versioneer.get_version(),
     cmdclass=versioneer.get_cmdclass(),
-    license='{{cookiecutter.open_source_license}}',
```

Comment:
We probably should keep this code path alive for when we relicense

Reply:
The jinja2 variables are the options that the users pick -- I'd rather put this back in when there's a choice to make.

Reply:
Fair enough - could this be raised in an issue re: updating the license once we go LGPL? just so we don't forget later

Reply:
Raised https://github.com/MDAnalysis/cookiecutter-mda/issues/12